op-challenger/cmd/main_test.go

```diff
@@ -995,6 +995,12 @@ func addRequiredArgsExcept(traceType types.TraceType, name string, optionalArgs
 	return append(toArgList(req), optionalArgs...)
 }
 
+func addRequiredArgsExceptMultiple(traceType []types.TraceType, name string, optionalArgs ...string) []string {
```

Comment:
I know it's long but it took me ages to work out what this was doing so maybe:
```suggestion
func addRequiredArgsForMultipleTracesExcept(traceType []types.TraceType, name string, optionalArgs ...string) []string {
```